src/node_modules/dbpedia/requests.mjs

```diff
@@ -54,3 +68,28 @@ export const getEntityDetails = async input => {
 
 	return filteredValues;
 };
+
+export const getClasses = async (input, depth = 'all') => {
+
+	const sanitizedURIs = sanitizeInput(input);
+	const queries = _.map(sanitizedURIs, URI =>
+		`{
+			BIND (${URI} as ?title)
+			OPTIONAL { ${URI} rdf:type ?type . }
+		}`,
+	);
+	const sparql = buildQuery(queries);
+	const results = await makeRequest(sparql);
+	const groups = _.group(results, _.getKey('title'));
+
+	const classFilter = new Set(await loadOntology(depth, { squash: true, fullURI: true }));
+	return _.mapValues(groups, group => {
+		const filtered = _.filter(group, item => classFilter.has(item.type));
+		return _.map(filtered, _.getKey('type'));
+	});
+};
+
+const main = async () => {
+	const results = await getClasses(['Earth', 'Moon'], 0);
+	console.log(results);
+};
```

Comment:
seems unused for now, I'd remove

Reply:
This is the logic we spoke of that allows us to retrieve classes for entities. I just have to update the `loadOntology` functionality but otherwise this is the function we'll use in the subsequent PR to populate the `ai_map` entities' classes and run data quality tests on them in `ai_map.spec.mjs`.

Reply:
I'd make a copy somewhere remove it (also cleaning up the code this depends on) as it's not in scope in this PR :)